packages/nodes-base/nodes/If/V2/IfV2.node.ts

```diff
@@ -13,6 +13,19 @@ import { ENABLE_LESS_STRICT_TYPE_VALIDATION } from '../../../utils/constants';
 import { looseTypeValidationProperty } from '../../../utils/descriptions';
 import { getTypeValidationParameter, getTypeValidationStrictness } from './utils';
 
+export const getOptions = (executeFunctions: IExecuteFunctions, itemIndex: number) => {
```

Comment:
nice extracting this code but unless it is needed in other places let's keep the functions private